tests/bud.bats

```diff
@@ -2422,15 +2431,15 @@ _EOF
 }
 
 @test "bud COPY with URL should fail" {
-  mkdir ${TESTSDIR}/bud/copy
-  FILE=${TESTSDIR}/bud/copy/Dockerfile.url
+  mkdir ${TESTDIR}/budcopy
+  FILE=${TESTDIR}/budcopy/Dockerfile.url
   /bin/cat <<EOM >$FILE
 FROM alpine:latest
 COPY https://getfedora.org/index.html .
 EOM
 
-  run_buildah 125 build --signature-policy ${TESTSDIR}/policy.json -t ${target} -f ${TESTSDIR}/bud/copy/Dockerfile.url ${TESTSDIR}/bud/copy
-  rm -r ${TESTSDIR}/bud/copy
+  run_buildah 125 build --signature-policy ${TESTSDIR}/policy.json -t foo -f ${TESTDIR}/budcopy/Dockerfile.url
+  expect_output --substring "error building .* source can.t be a URL for COPY"
```

Comment:
Should this be instead?
```suggestion
  expect_output --substring "error building .* source can't be a URL for COPY"
```

Reply:
(apostrophe instead of single wild-card character): I prefer not playing the shell quoting game, because that way lies madness.

Reply:
I agree.